packages/components/CHANGELOG-FIGMA-COMPONENTS.md

```diff
@@ -1,14 +1,239 @@
-# [HDS Components UI Kit](https://www.figma.com/design/iweq3r2Pi8xiJfD9e6lOhF/HDS-Components-v2.0?m=auto&node-id=2-7&t=YeAupEyKdQqauO1k-1)
+# [HDS Components UI Kit v2.0](https://www.figma.com/design/iweq3r2Pi8xiJfD9e6lOhF/HDS-Components-v2.0?m=auto&node-id=2-7&t=HYGTIoXBy2YkVWDP-1)
 
 ## December 20th, 2024
 
 `IconTile` - Removed the border and updated the colors for improved contrast and to create a distinctive look that aligns better with the surrounding elements.
 
-**Deprecated** HDS Components UI Kit v1.0. It will no longer receive updates or support. 
+`SideNav` - No longer deprecated due to adjustments in prioritization.
 
-Released HDS Components UI Kit v2.0.
+### Figma v2.0 Components
 
-`SideNav` - No longer deprecated due to adjustments in prioritization.
+Released HDS Components UI Kit v2.0. HDS Components UI Kit v1.0 is now deprecated and will no longer receive updates or support.
+
+!!! Insight
+
+**Migration Note**
+
+Many of the changes in the Figma v2.0 Library will result in a breaking change when replacing the v1.0 library components. For a comprehensive list of how these changes will impact your designs, refer to the [changelog document](https://docs.google.com/document/d/1q9IGtrsVOf3KrSBGb8AdF3OhPUm3fYAwPdhSml7onMM/edit?tab=t.0#heading=h.15f60fqiivvr).
+!!!
+
+#### General changes
+
+- Improved consistency in component naming conventions
+- Enhanced property panel customization options
+- Reduced variant complexity through boolean properties
+- Added local tokens for custom styling (hidden from publishing)
+- Standardized nested component structure
+
+#### Component organization updates
+
+- Form components now use "Form /" prefix instead of table of contents structure
+- Dialog-based components (Modal, Flyout) now use DialogPrimitive as base
+- Added [Template] components for common component combinations
+- Improved nested component exposure for easier customization
+- Standardized boolean property usage across components
+
+#### Component changes
```

Comment:
We may not need to reiterate the list of component changes if we link them to the more thorough document already.

Reply:
Just so I'm clear, are you suggesting we remove this entire section and just link to the changelog doc we aggregated?

Reply:
Yea, essentially pulling out the migration note at the top into plain text with the link to the file.

Reply:
Ok, I've done that, seems to be much simpler and is probably easier for consumers to digest if they really want to go deep into the changes (or find a specific one)